### PR TITLE
Mock list watchlist

### PR DESCRIPTION
**Pull request by Yuan-EPAM**
- implement endpoint responds with static data 
- have test endpoint with jest (expo-jest)

### CODE REVIEW

File: backend-opal/main.js [hunk 1]
```diff
@@ -2,14 +2,17 @@ require('dotenv').config();
 const express = require('express');
 
 const server = express();
+const PORT = 3001;
```
Comment (adamgyulavari) [inline backend-opal/main.js]: @Yuan-EPAM you should create a `.env` file and put the port there so you don't need to change the server.listen part

Comment (adamgyulavari) [reply backend-opal/main.js]: you can add these steps to the readme, so others can remember it too

File: frontend-opal/package.json [hunk 2]
```diff
@@ -16,7 +18,19 @@
...
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-prettier": "^6.5.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react": "^7.16.0",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
+    "supertest": "^4.0.2"
```
Comment (adamgyulavari) [inline frontend-opal/package.json]: @Yuan-EPAM why these are in this PR?

File: frontend-opal/.gitignore [hunk 3]
```diff
@@ -13,4 +13,5 @@ web-report/
 # macOS
 .DS_Store
 
+
 coverage/**/*
```
[the remark this answers is not in the document]
Comment (Yuan-EPAM) [reply frontend-opal/.gitignore]: The `coverage` folder is created by the `jest` automatically, which consists of some HTML reports and strange configuration files..

**Pull request merged**